queue.yaml

```diff
@@ -8,9 +8,24 @@ queue:
   rate: 2/m
 - name: emails
   rate: 5/s
+  retry_parameters:
+    task_retry_limit: 8
+    min_backoff_seconds: 0.1
+    max_backoff_seconds: 12.8
+    max_doublings: 8
```

Comment:
Where do these values come from? We should add context here, if possible, so that future maintainers understand how to specify these values.